drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableViewImpl.java

```diff
@@ -0,0 +1,640 @@
+                                                                           public void execute( final ActionCol52 column ) {
+                                                                               presenter.appendColumn( column );
+                                                                           }
+                                                                       },
+                                                                       column,
+                                                                       true,
+                                                                       !access.isEditable() );
+        popup.show();
+    }
+
+    @Override
+    public void newExtendedEntryActionSetColumn() {
+        final ActionSetFieldCol52 column = new ActionSetFieldCol52();
+        final ActionSetFieldPopup popup = new ActionSetFieldPopup( model,
+                                                                   oracle,
+                                                                   presenter,
+                                                                   new ActionColumnCommand() {
+                                                                       public void execute( final ActionCol52 column ) {
+                                                                           presenter.appendColumn( column );
+                                                                       }
+                                                                   },
+                                                                   column,
+                                                                   true,
+                                                                   !access.isEditable() );
+        popup.show();
+    }
+
+    @Override
+    public void newLimitedEntryActionSetColumn() {
+        final LimitedEntryActionSetFieldCol52 column = new LimitedEntryActionSetFieldCol52();
+        final ActionSetFieldPopup popup = new ActionSetFieldPopup( model,
+                                                                   oracle,
+                                                                   presenter,
+                                                                   new ActionColumnCommand() {
+                                                                       public void execute( final ActionCol52 column ) {
+                                                                           presenter.appendColumn( column );
+                                                                       }
+                                                                   },
+                                                                   column,
+                                                                   true,
+                                                                   !access.isEditable() );
+        popup.show();
+    }
```

Comment:
The two methods above are the same except for the first line...perhaps reuse the code instead?